src/Content/Text/BBCode.php

```diff
@@ -517,7 +517,7 @@ private static function convertUrlForActivityPub(string $url): string
 	 * @param string $url URL that is about to be reformatted
 	 * @return string reformatted link
 	 */
-	private static function getStyledURL(string $url): string
+	public static function getStyledURL(string $url): string
```

Comment:
Please move this method to the `Util\Strings` class now that it is public and not specific to BBCodes.

Reply:
Done